src/python/grapl-tests-common/grapl_tests_common/setup.py

```diff
@@ -109,16 +109,11 @@ def _after_tests() -> None:
 
 
 def exec_pytest() -> None:
-    pytest_args = []
+    pytest_args: List[Any] = []
```

Comment:
more accurately, this is a List[str], but whatevs

Reply:
fixed